msm/src/serialization/constraints.rs

```diff
@@ -63,12 +68,22 @@ impl<F: PrimeField> InterpreterEnv<F> for Env<F> {
         Column::X(3 + j)
     }
 
-    fn range_check15(&mut self, _value: &Self::Variable) {
-        // TODO
+    fn range_check15(&mut self, value: &Self::Variable) {
+        let one = ConstantExpr::from(ConstantTerm::Literal(F::one()));
+        self.rangecheck15_lookups.push(Lookup {
+            table_id: LookupTable::RangeCheck15,
+            numerator: Expr::Atom(ExprInner::Constant(one)),
```

Comment:
I must assume you only have "static" range checks? Like, they are always added, regardless of the values of other conditionals?

Reply:
Yes